app/src/main/java/com/example/concordia_campus_guide/view_models/PathsViewModel.java

```diff
@@ -18,6 +17,8 @@
 
 public class PathsViewModel extends ViewModel {
 
+    public static final int EARTH_RADIUS_KM = 6371;
```

Comment:
this is only used in one method no ? so why do we have  an attribute for it ? Wouldn't be better if we just keep inside the getDistanceFromLatLonInKm() method ?

Reply:
I think that constants should be as an attribute of the class where it is used

Reply:
I don't see why? since it is not used any other method in the class